src/transformer/transformer.js

```diff
@@ -71,7 +71,7 @@ export default async function (transformCallback, validateCallback) {
 
 			const records = await transform();
 			const failedRecords = records.filter(r => r.failed).map(result => {
-				return {...result, record: result.record.toObject()};
+				return {...result};
```

Comment:
The record needs to formatted with `toObject` to remove internal properties (`validationOptions`).

Anyway, this cannot be the problem with empty objects in the db, since the passed in value is still a object. I'd guess the problem lies within the API client module or the API itself.